service/project.go

```diff
@@ -76,15 +92,16 @@ func (s *ProjectService) Delete(ctx context.Context, projectID uuid.UUID, authUs
 		return err
 	}
 
-	_, err := s.Get(ctx, projectID, authUserID)
-	if err != nil {
+	if exists, err := s.Exists(ctx, projectID, authUserID); err != nil {
 		return err
```

Comment:
Since we are in a refactoring / cleanup phase, see if you can send an error from here in a following fashion:
`return fmt.Errorf("checking if project exists: %w", err)`

Reply:
Probably a separate PR, if we decide to invest in better "erroring".

Reply:
Created #25